test/scales/modifiedLogScaleTests.ts

```diff
@@ -2,187 +2,241 @@
 
-    it("autoDomain() expands single value correctly", () => {
-      scale.padProportion(0);
-      let singleValue = 15;
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [singleValue, singleValue]);
-      assert.deepEqual(scale.domain(), [singleValue / base, singleValue * base],
-        "positive single-value extent was expanded to [value / base, value * base]");
-      singleValue = -15;
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), [singleValue * base, singleValue / base],
-        "negative single-value extent was expanded to [value * base, value / base]");
-      singleValue = 0;
-      scale.autoDomain();
-      assert.deepEqual(scale.domain(), [-base, base],
-        "zero single-value extent was expanded to [base, -base]");
-    });
+      it("ensures x = invert(scale(x))", () => {
+        [0, 1, base, 100, 0.001, -1, -0.3, -base, base - 0.001].forEach((x) => {
+          assert.closeTo(x, scale.invert(scale.scale(x)), epsilon);
+          assert.closeTo(x, scale.scale(scale.invert(x)), epsilon);
+        });
+      });
 
-    it("domainMin()", () => {
-      let scale = new Plottable.Scales.ModifiedLog(base);
-      scale.padProportion(0);
-      let requestedDomain = [-5, 5];
-      scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => requestedDomain);
+      it("defaults to the [0, base] domain", () => {
+        assert.deepEqual(scale.domain(), [0, base], "default domain is [0, base]");
+      });
 
-      let minBelowBottom = -10;
-      scale.domainMin(minBelowBottom);
-      assert.deepEqual(scale.domain(), [minBelowBottom, requestedDomain[1]], "lower end of domain was set by domainMin()");
+      it("can be padded", () => {
+        scale.addIncludedValuesProvider((scale: Plottable.Scales.ModifiedLog) => [0, base]);
+        scale.padProportion(0);
+        let unpaddedDomain = scale.domain();
+        scale.padProportion(0.1);
+        assert.operator(scale.domain()[0], "<", unpaddedDomain[0], "left side of domain has been padded");
+        assert.operator(unpaddedDomain[1], "<", scale.domain()[1], "right side of domain has been padded");
+      });
 
-      let minInMiddle = 0;
-      scale.domainMin(minInMiddle);
-      assert.deepEqual(scale.domain(), [minInMiddle, requestedDomain[1]], "lower end was set even if requested value cuts off some data");
+      it("works on inverted domain", () => {
+        scale.domain([200, -100]);
+        scale.range([10, 20]);
+        let range = scale.range();
+        assert.strictEqual(scale.scale(-100), range[1], "minimum value in domain maps to maximum value in range");
+        assert.strictEqual(scale.scale(200), range[0], "maximum value in domain maps to minimum value in range");
+        let a = [-100, -10, -3, 0, 1, 3.64, 50, 60, 200];
+        let b = a.map((x) => scale.scale(x));
+        assert.deepEqual(b, b.slice().sort().reverse(), "should be decreasing function; reverse is sorted");
+      });
```

Comment:
That is a function of the scale itself. Tests that the scale does not blow up when the domain is reversed